cloud/jenkins/pxc_operator_gke_latest.groovy

```diff
@@ -166,32 +198,44 @@ void createCluster(String CLUSTER_SUFFIX) {
     withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-key-file', variable: 'CLIENT_SECRET_FILE')]) {
         sh """
             export KUBECONFIG=/tmp/$CLUSTER_NAME-$CLUSTER_SUFFIX
-
             maxRetries=15
             exitCode=1
+
             while [[ \$exitCode != 0 && \$maxRetries > 0 ]]; do
                 gcloud container clusters create $CLUSTER_NAME-$CLUSTER_SUFFIX \
+                    --release-channel $GKE_RELEASE_CHANNEL \
                     --zone $region \
-                    --cluster-version $USED_PLATFORM_VER \
+                    --cluster-version $PLATFORM_VER \
                     --preemptible \
                     --disk-size 30 \
                     --machine-type n1-standard-4 \
-                    --num-nodes=3 \
-                    --min-nodes=3 \
+                    --num-nodes=4 \
```

Comment:
Why 4?

Reply:
Because this is how we do it for PSMDB so I wanted to be the same for PXC.